packages/icons/CHANGELOG.md

```diff
@@ -25,3 +25,16 @@ sed -i '' 's#@goodhood/icons/lib/tiny/arrow_up#@goodhood/icons/lib/tiny/chevron_
 sed -i '' 's#@goodhood/icons/lib/tiny/arrow_left#@goodhood/icons/lib/tiny/chevron_left#g' **/*.jsx
 sed -i '' 's#@goodhood/icons/lib/tiny/arrow_right#@goodhood/icons/lib/tiny/chevron_right#g' **/*.jsx
 ```
+
+## v4.0.0 Changes
```

Comment:
I really like the trend of providing ready-to-use migration scripts.

Maybe make it work with `tsx` files as well?

Reply:
Done